Author: Muammar El Khatib <muammar@debian.org>
Description: Erasing the directory utils from clean target in Makefile.Linux
because this directory is not created in the build process. 
Index: makehuman-1.0.0~alpha6/Makefile.Linux
===================================================================
--- makehuman-1.0.0~alpha6.orig/Makefile.Linux	2011-03-01 02:24:41.000000000 +0100
+++ makehuman-1.0.0~alpha6/Makefile.Linux	2012-06-27 01:25:47.100802706 +0200
@@ -5,7 +5,7 @@
 
 CCOPTS = -Wall -O0 -g
 
-PYTHONVER     = 2.6
+PYTHONVER     = 2.7
 EXTRAFMWKS    = -lX11 -lGL -lGLU -lSDL -lGLEW
 EXTRALIBS     = -lpython$(PYTHONVER)
 EXTRADEFINES  =
@@ -13,7 +13,7 @@
 
 EXE = makehuman
 
-	
+
 %.o : %.c
 	@echo "compiling C file $< to $@ ..."
 	@gcc -c -MMD $(CCOPTS) $(INCLUDEPATH) $< -o $@
@@ -30,9 +30,9 @@
 	@echo "compiling ObjC++ file $< to $@ ..."
 	@g++ -c -MMD $(CCOPTS) $(INCLUDEPATH) $< -o $@
 
-$(EXE) : $(OBJS)	
+$(EXE) : $(OBJS)
 	@echo "linking as $@ ."
-	@g++ $(EXTRALIBS) $(EXTRAFMWKS) $(OBJS) -o $@
+	@g++ $(EXTRALIBS) $(EXTRAFMWKS) $(OBJS) -Wl,--as-needed -o $@
 
 # ---------------------------------------------------------------------
 # Cleanup unused stuff
@@ -40,7 +40,6 @@
 clean:
 	$(RM) $(OBJS) $(DEPS) *.obj *~ *.bak *%% *~ makehuman
 	find apps -name "*.pyc" -exec "rm" "-f" {} ";"
-	find utils -name "*.pyc" -exec "rm" "-f" {} ";"
 	find core -name "*.pyc" -exec "rm" "-f" {} ";"
 
 # ---------------------------------------------------------------------
